samples/highcharts/data-tools/csv-modifiers/demo.js

```diff
@@ -0,0 +1,242 @@
+const csvData = document.getElementById('csv').innerHTML;
+
+Highcharts.setOptions({
+    chart: {
+        zooming: {
+            enabled: true,
+            type: 'x'
+        }
+    },
+    yAxis: {
+        max: 5,
+        min: -1
+    },
+    xAxis: {
+
+        type: 'category'
+    },
+    tooltip: {
+        pointFormat: '{series.name} had <b>{point.y:,.2f}%</b><br/> population growth in {point.name}'
+    },
+    legend: {
+        enabled: false
+    },
+    plotOptions: {
+        series: {
+            marker: {
+                enabled: false
+            }
+        }
+    }
+});
+
+Dashboards.board('container', {
+    dataPool: {
+        connectors: [{
+            id: 'population-growth',
+            type: 'CSV',
+            options: {
+                csv: csvData,
+                firstColumnsAsNames: true,
+                dataTable: {
+                    aliases: {
+                        0: 'Brazil',
+                        1: 'Argentina',
+                        2: 'Uruguay',
+                        3: 'Paraguay',
+                        4: 'United States',
+                        5: 'Canada',
+                        6: 'Mexico',
+                        7: 'Guatemala',
+                        8: 'China',
+                        9: 'Japan',
+                        10: 'India',
+                        11: 'Indonesia'
+                    }
+                },
+                dataModifier: {
+                    type: 'Chain',
+                    chain: [{
+                        type: 'Invert'
+                    }, {
+                        type: 'Range',
+                        ranges: [{
+                            column: 'columnNames',
+                            minValue: '1961',
+                            maxValue: '2021'
+                        }]
+                    }]
+                }
+            }
+        }]
+    },
+    gui: {
+        layouts: [{
+            rows: [{
+                cells: [{
+                    id: 'south-america-chart',
+                    width: '1/3'
+                }, {
+                    width: '1/3',
+                    id: 'north-america-chart'
+                }, {
+                    width: '1/3',
+                    id: 'asia-chart'
+                }]
+            }, {
+                cells: [{
+                    id: 'legend'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        cell: 'asia-chart',
+        sync: {
+            visibility: true,
+            extremes: true
```

Comment:
Do we need extremes sync here? It only causes issues with the `reset zoom` button.

Reply:
PS I reported this issue on Assana 😄